HLTriggerOffline/Btag/interface/HLTBTagHarvestingAnalyzer.h

```diff
@@ -48,6 +49,8 @@ class HLTBTagHarvestingAnalyzer : public DQMEDHarvester {
 			// Histogram handler
 			std::map<std::string, MonitorElement *> H1_;
 
+			std::vector<std::string> modules_;
```

Comment:
Right, we agree to monitor the plots in these three slices. The histograms are filled and put to the corresponding folder named "HEP17" etc.